zboxcore/sdk/test/common/allocation.json

```diff
@@ -0,0 +1,124 @@
+{
```

Comment:
best to put the json files on a `testdata` folder designed to keep test files on go.

https://golang.org/pkg/cmd/go/internal/test/

```
The go tool will ignore a directory named "testdata", making it available
to hold ancillary data needed by the tests.
```